src/mlpack/core/optimizers/sga/sga_update.hpp

```diff
@@ -25,7 +25,7 @@ namespace optimization {
                 Function Approximation },
  *   journal = {NIPS},
  *   year    = {2000},
- *   url     = {https://goo.gl/UwfMnd}
+ *   url     = {https://goo.gl/2Gtmqu}
```

Comment:
I don't know whether we allow google shorteners or not. I was searching for it in the design wiki. I think @zoq @rcurtin would give you better idea on that.

Reply:
Dear @zoq , @rcurtin The link to the paper is long. The link is https://papers.nips.cc/paper/1713-policy-gradient-methods-for-reinforcement-learning-with-function-approximation.pdf .Currently I am using a google shortner.  How should I reference it here . Thanks